maestro/src/agents/remote_agent.py

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+
+import os, dotenv
+import asyncio
+import requests
+from requests import RequestException
+
+from src.agents.agent import Agent
+
+dotenv.load_dotenv()
+
+class RemoteAgent(Agent):
+    """
+    RemoteAgent extends the Agent class to load and run a specific agent.
+    """
+
+    def __init__(self, agent: dict) -> None:
+        """
+        Initializes the workflow for the specified agent.
+
+        Args:
+            agent_name (str): The name of the agent.
+        """
+        super().__init__(agent)
+
+
+    async def run(self, prompt: str) -> str:
+        """
+        Runs the agent with the given prompt.
+        Args:
+            prompt (str): The prompt to run the agent with.
+        """
+        print(f"🐝 Running {self.agent_name}...\n")
```

Comment:
Can we use a different emoji for remote agent? Trying to have the emoji match the type of agents. So perhaps 👩🏻‍💻 or 💸

Reply:
And let's make it consistent